flow_apollo2_data_processing/processing/flatfile-to-json.cwl

```diff
@@ -8,7 +8,8 @@ requirements:
   - class: InitialWorkDirRequirement
     listing: 
       ${
-        var LIST = [(inputs.in_gff)];
+        var LIST = [(inputs.in_gff),
+                   (inputs.in_data)];
```

Comment:
Let folder "data" become a staging input, which means that a symbolic link of folder "data" will be in the tmp folder before the base command start.